tools/prowgen/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.19 as build-env
+FROM golang:1.20 as build-env
```

Comment:
Is there a reason to not go to 1.21 here? I know 1.19 is EOL and 1.20 should be EOL soon as well.

Reply:
It's not really used for anything. It's only used when we are debugging prowgen anyway.